pkg/azurefile/controllerserver.go

```diff
@@ -312,6 +312,16 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Errorf(codes.InvalidArgument, "fsType(%s) is not supported with protocol(%s)", fsType, protocol)
 	}
 
+	if account != "" && sku == "" {
```

Comment:
also add `fileShareSize < minimumPremiumShareSize` check, that could save one `StorageAccountClient.GetProperties` call

Reply:
Added, along with one more unit test to verify `StorageAccountClient.GetProperties` is not called in this case.